### ISSUE TITLE
Model details

### ISSUE THREAD

**Issue by Pan-yangxu** [suno-ai/bark#302]
hello，about the model training
can I ask some questions
is the text model's inputs is the token ids which is encoded by bert model from text, and the output is also the text token ids? 
if this，here i find the input_vocab_size is not equal with output_vocab_size, which i can not understand

        assert config.input_vocab_size is not None  #129600
        assert config.output_vocab_size is not None  #10048
        assert config.block_size is not None        #1024
        self.config = config

        self.transformer = nn.ModuleDict(dict(
            wte = nn.Embedding(config.input_vocab_size, config.n_embd),
            wpe = nn.Embedding(config.block_size, config.n_embd),
            drop = nn.Dropout(config.dropout),
            h = nn.ModuleList([Block(config, idx) for idx in range(config.n_layer)]), #24
            ln_f = LayerNorm(config.n_embd, bias=config.bias),  #1024
        ))
        self.lm_head = nn.Linear(config.n_embd, config.output_vocab_size, bias=False)

second：in generate_coarse
 semantic_to_coarse_ratio = COARSE_RATE_HZ / SEMANTIC_RATE_HZ * N_COARSE_CODEBOOKS
How did this parameter “SEMANTIC_RATE_HZ ” come about?

looking forward to reply，thanks
